frontend/src/store/combat/index.ts

```diff
@@ -69,6 +69,15 @@ const combat = {
       const { CryptoBlades } = rootState.contracts();
       if(!CryptoBlades || !rootState.defaultAccount) return;
 
+      return await CryptoBlades.methods
+        .vars(28)
```

Comment:
Magic number!

Reply:
Yes it is, but less contract calls. We still have tons of errors on loading and I believe extra calls don't help with that.

Reply:
No. You can just do `const MEANINGFUL_NAME = 28;`, no need to fetch from the contract